jenkins/automation/alex.groovy

```diff
@@ -395,6 +395,22 @@ pipeline {
                     to: useEmailList
                 }
             }
+        }
+	    stage('Send Email to cortx-cc') {
```

Comment:
@mukul-seagate11  https://github.com/Seagate/cortx-cc repo is private as of now. Hope it will not cause any issues in Alex report generation and sending email notification

Reply:
No, as its already tested based on https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/Subha_WS/job/alex-test/70/console